js/components/controlBar.js

```diff
@@ -359,32 +358,16 @@ var ControlBar = createReactClass({
       }
     }
 
-    var totalTime = 0;
-    if (
-      this.props.duration === null ||
-      typeof this.props.duration === 'undefined' ||
-      this.props.duration === ''
-    ) {
-      totalTime = Utils.formatSeconds(0);
-    } else {
-      totalTime = Utils.formatSeconds(this.props.duration);
-    }
+    var totalTime = this.props.getTotalTime();
```

Comment:
Is it necessary to call a function in this case or could the formatted value be simply passed as a prop?